external-import/recorded-future/src/rflib/rf_alerts.py

```diff
@@ -1,389 +1,441 @@
+import base64
 import datetime
 import threading
-from os import path
 from re import search
-from sys import exit
-from time import sleep
 
 import pycti
 import pytz
-import yaml
+import stix2
+import tldextract
+from pycti import Incident
 
+from .constants import TLP_MAP
+from .make_markdown_table import make_markdown_table
 from .pyrf import RecordedFutureApiClient
 
 
+@stix2.CustomObject(
+    "fakechannelbecauseofbug",
+    [
+        ("name", stix2.properties.StringProperty(required=True)),
+        ("description", stix2.properties.StringProperty()),
+        (
+            "channel_types",
+            stix2.properties.ListProperty(contained=stix2.properties.StringProperty()),
+        ),
+        ("object_marking_refs", stix2.properties.STIXObjectProperty()),
+        ("createdBy", stix2.properties.StringProperty()),
+        ("created", stix2.properties.StringProperty()),
+    ],
+)
+class OpenCTIChannel:
+    pass
+
+
+@stix2.CustomObject(
```

Comment:
Why redefine the CustomObservableText object here and not use the one in pcyti?
https://github.com/OpenCTI-Platform/client-python/blob/master/pycti/utils/constants.py#L268